packages/components/src/components/Dropzone/Dropzone.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import type { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import { FileWithPath } from "react-dropzone";
+import { Dropzone } from "./Dropzone";
+
+const resolveAfter2Seconds = () => {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      resolve("Document uploaded successfully");
+    }, 2000);
+  });
+};
+
+const sendDocument = async (
+  file: FileWithPath,
```

Comment:
What do you think about exposing our own type? I think this way people will always import something from lib we are using. Instead the could import something like: 

```js
import { Dropzone, FileWithPath } from "./Dropzone";
```